lib/administrate/fields/base.rb

```diff
@@ -16,6 +16,14 @@ def self.searchable?
         false
       end
 
+      def self.search_predicate(attribute, _term)
+        "lower(#{attribute}) LIKE ?"
```

Comment:
What happens if we return the string `"false"` here?

I'm just guessing, but that might mean we can get rid of the `self.searchable?` method. The `false` would get inserted into the search string as something like this:

```sql
WHERE LOWER(string_field) LIKE search_term OR false OR enum_field = 1
```

The `OR false` clause wouldn't have any effect on the query.

We could move this search predicate definition into `Administrate::Field::String`, to keep that working.

As for the search term, we could change it to `nil`, and then run `.uniq` on the array of search terms to remove nils before passing it into the search.

That's all a guess, but it would be pretty cool if it worked. What do you think of that approach? There may be downsides that I've missed.